app/services/user_svc.py

```diff
@@ -19,8 +19,8 @@ def signup():
             db.session.add(user)
             db.session.commit()
             flash(f'Welcome {user.username}! Please login.', 'notify')
-            # lform.username.data = user.username
-            return redirect(url_for('main_bp.dashboard'))
+            lform.username.data = user.username
```

Comment:
I completely forgot to ask about this awhile back. I remember now this line of code was commented out for some reason. Good job @Xuehong-pdx !